recipes/bayestyper/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "1.5" %}
+
+package:
+  name: bayestyper
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  url: https://github.com/bioinformatics-centre/BayesTyper/archive/v{{ version }}.tar.gz
+  sha256: 917cd1b1ca7b5cfb6b8327138515ca1ad70878f0c4e8df393fddcbe42f281e14
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - boost
```

Comment:
Can you change this to `boost-cpp`? `boost` is actually a python interface that happens to pull an older `boost-cpp` in.

Reply:
Done. Let's see how it goes.